### ISSUE TITLE
confused on how to install the cursors

### ISSUE THREAD

**Issue by ivbeenmemed** [vinceliuice/Qogir-icon-theme#83]
after installing the icon theme i tried to enable the cursor themes, however they just put on the normal DMZ cursors. Did i install it wrong? or is this a bug?

**Comment by vinceliuice** [comment]
It should not be, the cursor theme is inside icon theme

**Comment by ivbeenmemed** [comment]
yeah i can see that, when i set the qogir cursors with gnome tweak tool it just sets the default dmz white cursor, for all variants
